recipes/md4qt/all/conanfile.py

```diff
@@ -14,12 +15,17 @@ class Md4QtConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/igormironchik/md4qt"
     license = "MIT"
-    description = "Header-only C++ library for parsing Markdown."
-    topics = ("markdown", "gfm", "parser", "icu", "ast", "commonmark", "md", "qt6", "stl", "cpp17")
-    package_type = "header-library"
+    description = "C++ library for parsing Markdown."
+    topics = ("markdown", "gfm", "parser", "ast", "commonmark", "md", "qt6", "cpp17")
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    def init(self):
```

Comment:
```suggestion
    def configure(self):
```
I'm not sure why, but it looks like `init` is not called, this is why conan does not consider the package as a header-library, so this is why ICU headers are not found
`configure(self)` is traditionnaly used for this : https://github.com/search?q=repo%3Aconan-io%2Fconan-center-index+%22self.package_type+%3D+%22&type=code

Reply:
It solves with version < 5.0.0, but with > 5.0.0 I have an issue. You will see soon, I've made a commit.